src/Aquifer.Jobs/Aquifer.Jobs.csproj

```diff
@@ -20,7 +20,9 @@
         </PackageReference>
 
     </ItemGroup>
-
+    <ItemGroup>
+        <PackageReference Include="Microsoft.ApplicationInsights.WorkerService" Version="2.22.0" />
+    </ItemGroup>
```

Comment:
It's weird that this created a new ItemGroup instead of putting it in the one above that has the package references already.

Reply:
I added the package wrong.

Reply:
pushed a fix